cassandra/data/benchflow.cql

```diff
@@ -57,7 +58,7 @@ CREATE TABLE environment_data (
   network_interfaces map<text, uuid>,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((experiment_id, trial_id), container_id, host_id, environment_data_id)
+  PRIMARY KEY ((experiment_id, trial_id), host_id, container_id, environment_data_id)
```

Comment:
@VincenzoFerme We are already partitioning by trial id and experiment id, and clustering by container and host. The data are already spatially located closed to each other at this point, adding container name to it is redundant at this point, and we are just increasing the primary key size for little gain (plus, primary keys shouldn't be made too large if possible)